lib/debugger/src/i.erl

```diff
@@ -81,17 +81,19 @@ im() ->
 -doc """
 Interprets the specified module(s) on the local node.
 
+- If `AbsModule :: Module | File`, then `Result :: {module, Module} | error`.
+- If `AbsModules :: [AbsModule]`, then `Result :: ok`.
+
 See `int:i/1` for more information.
 """.
--spec ii(AbsModule) -> {module, Module} | error when
-      AbsModule :: Module | File,
-      Module :: module(),
-      File :: file:name_all();
-        (AbsModules) -> ok when
-      AbsModules :: [AbsModule],
-      AbsModule :: Module | File,
-      Module :: module(),
-      File :: file:name_all().
+-spec ii(AbsModules | AbsModule) -> Result when
+          AbsModules :: [AbsModule],
+          AbsModule :: Module | File,
```

Comment:
The main issue here is that File can be a list of characters, which makes the empty list overlap with the empty list of AbsModules. For now, I think the best we can do is merge the overloaded function specification and add documentation that explains the link between input and output

Reply:
Can you change `[AbsModule]` to `[AbsModule,...]` ?

Reply:
The implementation calls `int:i/1` which calls `int:i2/3` which allows for an empty list, but the type suggested here should forbid passing empty list.

Is it a failure to pass empty list? At the moment, that returns `ok`.
If we agree on not allowing empty list, should I also fix other types in near modules such as `debugger:ini/2`?